cmd/expiration-mailer/main.go

```diff
@@ -247,7 +247,7 @@ func main() {
 		tmpl, err := template.New("expiry-email").Parse(string(emailTmpl))
 		cmd.FailOnError(err, "Could not parse email template")
 
-		mailClient := mail.New(c.Mailer.Server, c.Mailer.Port, c.Mailer.Username, c.Mailer.Password)
+		mailClient := mail.New(c.Mailer.Server, c.Mailer.Port, c.Mailer.Username, c.Mailer.Password, c.Mailer.From)
```

Comment:
Okay, this is a bit easier to see now. Sorry. I think we need a `smtp.ParseAddress(c.Mailer.From)` just to make sure the From line is well-formed.